promql/testdata/trig_functions.test

```diff
@@ -0,0 +1,71 @@
+# Testing sin() cos() tan() asin() acos() atan() rad() deg() pi().
+
+load 5m
+	trig{l="x"} 10
+	trig{l="y"} 20
+	trig{l="NaN"} NaN
+
+eval instant at 5m sin(trig)
+	{l="x"} -0.5440211108893699
+	{l="y"} 0.9129452507276277
+	{l="NaN"} NaN
+
+eval instant at 5m cos(trig)
+	{l="x"} -0.8390715290764524
+	{l="y"} 0.40808206181339196
+	{l="NaN"} NaN
+
+eval instant at 5m tan(trig)
+	{l="x"} 0.6483608274590867
+	{l="y"} 2.2371609442247427
+	{l="NaN"} NaN
+
+eval instant at 5m asin(trig - 10.1)
+  {l="x"} -0.10016742116155944
+  {l="y"} NaN
+	{l="NaN"} NaN
```

Comment:
Something ended up inconsistently indented here and in other tests below? :thinking:

Reply:
Whoops, we're doing tabs, not spaces.